schemas/json/layout/layout.schema.v1.json

```diff
@@ -0,0 +1,213 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Altinn layout",
+    "description": "Schema that describes the layout configuration for Altinn applications.",
+    "type": "object",
+    "additionalProperties": true,
+    "properties": {
+      "data": {
+        "$ref": "#/definitions/data"
+      }
+    },
+    "definitions": {
+      "data": {
+        "title": "data",
+        "description": "Contains data describing the layout configuration.",
+        "type": "object",
+        "properties": {
+          "layout": {
+            "$ref": "#/definitions/layout"
+          }
+        }
+      },
+      "layout": {
+        "title": "layout",
+        "description": "Contain components to be included in the layout.",
+        "type": "array",
+        "items": {
+          "$ref": "#/definitions/component"
+        }
+      },
+      "component": {
+        "type": "object",
+        "properties": {
+          "id": {
+            "type": "string",
+            "title": "id",
+            "description": "The component ID. Must be unique within a given layout."
+          },
+          "type": {
+            "type": "string",
+            "title": "type",
+            "description": "The component type.",
+            "enum": ["AddressComponent", "Paragraph", "Header", "Input", "TextArea", "Checkboxes", "RadioButtons", "Dropdown", "FileUpload", "Datepicker", "NavigationButtons", "Button", "Group"]
+          },
+          "required": {
+            "type": "boolean",
+            "title": "required",
+            "description": "Boolean value indicating if the component is required when filling in the form. Defaults to false."
```

Comment:
Add `default` for all properties with default values. https://json-schema.org/understanding-json-schema/reference/generic.html#annotations